drivers/storage/portworx/podsecuritypolicies.go

```diff
@@ -0,0 +1,96 @@
+package portworx
+
+import (
+	"github.com/libopenstorage/operator/pkg/constants"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func portworxPodSecurityPolicies() []policyv1beta1.PodSecurityPolicy {
+	return []policyv1beta1.PodSecurityPolicy{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: constants.PrivilegedPSPName,
+			},
+			Spec: policyv1beta1.PodSecurityPolicySpec{
+				Privileged:             true,
+				ReadOnlyRootFilesystem: true,
+				HostNetwork:            true,
+				HostIPC:                true,
+				HostPID:                true,
+				HostPorts: []policyv1beta1.HostPortRange{
+					{Min: 0, Max: 65535},
+				},
+				Volumes: []policyv1beta1.FSType{policyv1beta1.All},
+				RunAsUser: policyv1beta1.RunAsUserStrategyOptions{
+					Rule: policyv1beta1.RunAsUserStrategyRunAsAny,
+				},
+				FSGroup: policyv1beta1.FSGroupStrategyOptions{
+					Rule: policyv1beta1.FSGroupStrategyRunAsAny,
+				},
+				SELinux: policyv1beta1.SELinuxStrategyOptions{
+					Rule: policyv1beta1.SELinuxStrategyRunAsAny,
+				},
+				SupplementalGroups: policyv1beta1.SupplementalGroupsStrategyOptions{
+					Rule: policyv1beta1.SupplementalGroupsStrategyRunAsAny,
+				},
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: constants.RestrictedPSPName,
+			},
+			Spec: policyv1beta1.PodSecurityPolicySpec{
+				Privileged:             false,
+				ReadOnlyRootFilesystem: true,
+				Volumes: []policyv1beta1.FSType{
+					policyv1beta1.ConfigMap,
+					policyv1beta1.Secret,
+					policyv1beta1.HostPath,
+					policyv1beta1.EmptyDir,
+				},
+				RunAsUser: policyv1beta1.RunAsUserStrategyOptions{
+					// TODO: update docker images to avoid using root user
+					Rule: policyv1beta1.RunAsUserStrategyRunAsAny,
+				},
+				FSGroup: policyv1beta1.FSGroupStrategyOptions{
+					Rule: policyv1beta1.FSGroupStrategyRunAsAny,
+				},
+				SELinux: policyv1beta1.SELinuxStrategyOptions{
+					Rule: policyv1beta1.SELinuxStrategyRunAsAny,
+				},
+				SupplementalGroups: policyv1beta1.SupplementalGroupsStrategyOptions{
+					Rule: policyv1beta1.SupplementalGroupsStrategyRunAsAny,
+				},
+			},
+		},
+		{
```

Comment:
We should create this only if Lighthouse is enabled. Nowadays, we mostly don't run Lighthouse so don't want an extra PSP created when LH is not being used.

Reply:
Removing psp for lighthouse.